webpack/components/QuestionWidget.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+var routes = require("../routes.js");
+if(routes.Routes){ routes=routes.Routes;}
```

Comment:
Though looking at the other comments maybe the import will resolve this, have to check and see